modules/regular_service_perimeter/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "policy" {
+  description = "Name of the parent policy"
+}
+
+variable "description" {
+  description = "Description of the regular perimeter"
+}
+
+variable "perimeter_name" {
+  description = "Name of the perimeter. Should be one unified string. Must only be letters, numbers and underscores"
+}
+
+variable "restricted_services" {
+  description = "GCP services that are subject to the Service Perimeter restrictions. Must contain a list of services. For example, if storage.googleapis.com is specified, access to the storage buckets inside the perimeter must meet the perimeter's access restrictions."
+  type        = "list"
+  default     = [""]
```

Comment:
Is this supposed to default to an empty list? Defaulting to a list with an empty string has caused us problems before.

Reply:
Yeah that's what it's supposed to be.
I fixed to default to an actual empty list here:
 5790fb66d98f4b022f8ae503458dd4b9d329b33d